akka-http-core/src/main/scala/akka/http/scaladsl/model/Trailer.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.scaladsl.model
+
+import akka.annotation.ApiMayChange
+import akka.http.javadsl.{ model => jm }
+
+import scala.collection.immutable
+
+class Trailer(@ApiMayChange val headers: immutable.Seq[(String, String)]) extends jm.Trailer {
+  /**
+   * Java API
+   */
+  override def addHeader(header: jm.HttpHeader): Trailer = {
+    val sheader = header.asInstanceOf[HttpHeader]
+    new Trailer((sheader.name, sheader.value) +: headers)
```

Comment:
Why to the front?

Reply:
No real reason - habit because it is faster for lists, but that likely won't be relevant here. I'll change it to append.